### PR TITLE
Implement full JSON support for group_vars

### PR DESCRIPTION
**Pull request by titom73**
## Description

Update AVD to support JSON as input structure as it is fully supported by Ansible.

Fix issue #180

## Changes

- Update VLANS and SVI templating engine to force
- Add a testing scenario to validate futur changes

### CODE REVIEW

**Comment by carlbuchmann** [comment]
@titom73 & @ankudinov - I agree with the proposed changes, this is also related to issue #140 raised by @ankudinov , is there other changes required?

**Comment by titom73** [comment]
PR only provides a fix by protecting group_vars value and force type for everything coming from JSON data: Keys from dictionary where `int` is expected.

I think #140 would require more work as we have to go through all templates to cast all variables to `int` when required.

File: ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/evpn-fabric-l3leaf-yml.j2 [hunk 1]
```diff
@@ -377,4 +377,4 @@ queue_monitor_streaming:
 {# management api http #}
 ### Management API http ###
 management_api_http:
-{% include 'base/management-api-http.j2' %}
\ No newline at end of file
+{% include 'base/management-api-http.j2' %} #}
```
Comment (carlbuchmann) [inline ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/evpn-fabric-l3leaf-yml.j2]: @titom73 please remove "#}"

**Review by carlbuchmann — CHANGES_REQUESTED** [review]

**Review by carlbuchmann — APPROVED** [review]
LGTM and tested with AVD Example

**Pull request merged**